apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-autocomplete-block-search-term.js

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+/**
+ * External dependencies
+ */
+import { debounce, get } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { select } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import tracksRecordEvent from './track-record-event';
+
+const trackAutocompleteBlockTerm = () => {
+	const search_term = get( select( 'core/block-editor' ).getSelectedBlock(), [
```

Comment:
> Can I confirm we need to do it like this because the Block's content is the "search box"?

Yes... 🤔 

> If so could we add a comment to clarify this?

For sure